internal/guides/generated/producer/setters.go

```diff
@@ -15,7 +15,15 @@
 // Code generated by "mdtogo"; DO NOT EDIT.
 package producer
 
-var SettersGuide = `Setters provide a solution for template-free setting or substitution of field
+var SettersGuide = `
+{{% hide %}}
```

Comment:
So there is a problem here. The guides we generate for the CLI includes the mdrip metadata. We should remove the html comments metadata that names the mdrip steps and remove everything between the `{{% hide %}}` blocks